Xamarin.Forms.Core/DecorableTextElement.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Xamarin.Forms
+{
+	[Flags]
+	[TypeConverter(typeof(TextDecorationConverter))]
+	public enum TextDecorations
+	{
+		None = 0,
+		Strikethrough = 1 << 1,
+		Underline = 1 << 0
```

Comment:
is this intended order?

Reply:
I usually alphabetize them, and the values are to match their values on UWP, but I can shift them if desired.

Reply:
I don't think alphabetical order is required for enum.  There is no such requirement and it is difficult to find such an example. Just like UWP's TextDecorations, it would be better to shift `Strikethrough` and `Underline`.

Reply:
I'd say shift them. Mainly because all the other [FLAGS]  are in numerical order

Reply:
If you want alphabetical order, how about exchange values?
``` C#
 None = 0,
 Strikethrough = 1 << 0,
 Underline = 1 << 1
```